plugins/slipping-away-users/frontend/public/javascripts/countly.models.js

```diff
@@ -1,30 +1,106 @@
-/*global countlyCommon,jQuery*/
-(function(countlySlippingPlugin, $) {
-    var _data = {};
-    /**
-     * This is for initializing model
-     * @param {object} query user drill filter query
-     * @return {func} ajax func to request data and store in _data
-    */
-    countlySlippingPlugin.initialize = function(query) {
+/*global countlyCommon,jQuery,countlyVue*/
+(function(countlySlippingAwayUsers, $) {
 
-        //returning promise
-        return $.ajax({
-            type: "GET",
-            url: countlyCommon.API_URL + "/o/slipping",
-            data: {
-                app_id: countlyCommon.ACTIVE_APP_ID,
-                method: 'slipping',
-                query: JSON.stringify(query),
-            },
-            success: function(json) {
-                _data = json;
+    countlySlippingAwayUsers.helpers = {
+        buildFilters: function(filters) {
```

Comment:
For now, this could stay. But in the future we might need to transfer this functionality to query builder itself.

Reply:
Assuming that other endpoints will want to send the same query format then I agree it should move inside query billder (I left a note).